src/ActivityFeeds/PostsFeedControls.jsx

```diff
@@ -105,54 +116,50 @@ const createQuery = (type) => {
     case "recentcommentdesc":
       querySortOption = `{ last_comment_timestamp: desc_nulls_last },`;
       break;
-    // More options...
     default:
       querySortOption = "";
   }
 
   let queryFilter = "";
   switch (type) {
     case "following":
+      let filteredAccountsFollowing = accountsFollowing;
+      if(filteredAccountIds) {
+        const filteredAccountList = filteredAccountIds.split(",");
```

Comment:
Nothing is using the filtering of the following list yet. I put it in for consistency in case groups or other future features both filter and use this feed.